extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerService.ts

```diff
@@ -4,30 +4,55 @@
  *--------------------------------------------------------------------------------------------*/
 
 
-import { ResourceServiceBase, GraphData } from '../resourceTreeDataProviderBase';
-import { azureResource } from 'azurecore';
+import { ServiceClientCredentials } from '@azure/ms-rest-js';
+import { ResourceGraphClient } from '@azure/arm-resourcegraph';
+import { ResourceServiceBase, GraphData, queryGraphResources } from '../resourceTreeDataProviderBase';
+import { azureResource, AzureAccount } from 'azurecore';
+import { serversQuery, synapseQuery } from './serverQueryStrings';
 
 export interface DbServerGraphData extends GraphData {
 	properties: {
 		fullyQualifiedDomainName: string;
 		administratorLogin: string;
+		connectivityEndpoints?: { sql: string };
+		managedResourceGroupName?: string;
+		sqlAdministratorLogin?: string;
 	};
 }
 
-export const serversQuery = `where type == "${azureResource.AzureResourceType.sqlServer}"`;
-
 export class AzureResourceDatabaseServerService extends ResourceServiceBase<DbServerGraphData, azureResource.AzureResourceDatabaseServer> {
 
 	protected get query(): string {
 		return serversQuery;
 	}
 
+	public override async getResources(subscriptions: azureResource.AzureResourceSubscription[], credential: ServiceClientCredentials, account: AzureAccount): Promise<azureResource.AzureResourceDatabaseServer[]> {
```

Comment:
I'd suggest just replacing the `extends ResourceServiceBase` and have it directly implement IAzureResourceService. With this override there's nothing from the base class you're actually using so it's just unnecessary overhead. 

(this is what the databaseService is doing, use that as guidance)